assets/js/ui/stations-observations/stations-list-page-header-map.js

```diff
@@ -0,0 +1,60 @@
+import domready from "mf-js/modules/dom/ready";
+import {createMap} from "../create-map";
+
+const retrieveStations = (map, query) => {
+    let url = dataRoute;
+    console.log(query);
+
+    if (!!query && 'object' == typeof JSON.parse(query)) {
+        url += '?';
+        for (const [key, value] of Object.entries(JSON.parse(query))) {
```

Comment:
for...of loop seems not really suitable here : query has only one line. And if query get more lines, then below url would be invalid as there is no '&' separator